apollo-training/src/lib/constant.js

```diff
@@ -0,0 +1,7 @@
+export default {
+  subscriptions: {
+    TRAINEE_ADDED: 'TRAINEE_ADDED',
+    TRAINEE_UPDATED: 'TRAINEE_UPDATED',
+    TRAINEE_DELETED: 'TRAINEE_DELETED',
```

Comment:
What are these variables.

Reply:
These are constant variable, to avoid self declaration in every i prefer to kept in  a separate file .